source/features/search-releases.tsx

```diff
@@ -0,0 +1,155 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+
+import features from '../libs/features';
+import {isReleasesOrTags} from '../libs/page-detect';
+import {getOwnerAndRepo} from '../libs/utils';
+import {octoface} from '../libs/icons';
+
+async function init() {
+	if (!isReleasesOrTags()) {
+		return false;
+	}
+
+	if (!isOnReleasePage()) {
```

Comment:
You are right but here we are already checking the page by `features.isReleasesOrTags`. I have added this check for the cases when a release is not present. e.g. release page of [refined-github](https://github.com/sindresorhus/refined-github/releases). I think the name is little ambiguos. Changing it to `hasAnyReleases`. What do you think?

Reply:
Gotcha, if it’s just checking if any releases exist at all (so “the feature isn’t needed” rather than “we’re not on the right page”) then it’s fine here in init. 

You can probably also inline a `select.exists` that looks for the “there are no releases” element. I think GitHub calls them “blankslate”

Reply:
I think looking for `.release` and `.release-entry` will suffice. No need for an extra check of `.blankslate`.

Reply:
It's not an extra check, but a single check for one element. Also generally you can combine selectors like `select.exists('.release, .release-entry')`

Reply:
Okay. Suggested included. 😅